script/cibuild

```diff
@@ -11,4 +11,4 @@ my @args = ("-workspace", $WORKSPACE, "-scheme", $SCHEME, $ACTION, "CODE_SIGNING
 unshift @args, "-sdk", "iphonesimulator" if $buildSettings =~ /\bPLATFORM_NAME = iphoneos/i;
 
 print "xcodebuild @args\n";
-exec("xcodebuild", @args);
+exec("xcodebuild @args");
```

Comment:
https://github.com/mdiep/Tentacle/pull/37#issuecomment-234604638: I'm not good at Perl and not sure if this is a right way to handle `XCODE_ACTION="build-for-testing test-without-building"` (`$ACTION`).